### PR TITLE
authenticate: add more verbose errors to authenticate handler

### PR DESCRIPTION
**Pull request by benny-gold**
Reasonably lazy PR to add some more info on which particular param is invalid - trying to troubleshoot my setup and struggling.  

**Checklist**:
- [👎] updated docs
- [🤠] unit tests added
- [🤡] related issues referenced
- [💯] ready for review

### CODE REVIEW

File: authenticate/handlers.go [hunk 1]
```diff
@@ -210,21 +210,21 @@ func (a *Authenticate) OAuthStart(w http.ResponseWriter, r *http.Request) {
...
+		httputil.ErrorResponse(w, r, "Invalid redirect parameter: proxy url not from the root domain", http.StatusBadRequest)
 		return
 	}
 
 	// get the signature and timestamp values then compare hmac
 	proxyRedirectSig := authRedirectURL.Query().Get("sig")
 	ts := authRedirectURL.Query().Get("ts")
 	if !middleware.ValidSignature(proxyRedirectURL.String(), proxyRedirectSig, ts, a.SharedKey) {
-		httputil.ErrorResponse(w, r, "Invalid redirect parameter", http.StatusBadRequest)
+		httputil.ErrorResponse(w, r, "Invalid redirect parameter: Invalid signature", http.StatusBadRequest)
```
Comment (desimone) [inline authenticate/handlers.go]: `Invalid` -> `invalid` just to be consistent

**Comment by desimone** [comment]
@benny-gold thanks for making the changes.

`git rebase -i master` and squash all but the top commit.  If you don't care about commit history, I can squash on my end but it will make it look like I made the changes.

**Comment by benny-gold** [comment]
Thanks @desimone - squashed on my end and kept forgetting force push! Muscle memory ;)

**Comment by desimone** [comment]
@benny-gold thank you!

**Pull request merged**